atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionTest.java

```diff
@@ -698,12 +707,12 @@ public void testColumnRangePagingTransaction_batchingVisitable() {
         verifyMatchingResult(expected, row, columnRange);
 
         columnRange =
-                t.getRowsColumnRange(TEST_TABLE, ImmutableList.of(row), BatchColumnRangeSelection.create(PtBytes.toBytes("col"), PtBytes.EMPTY_BYTE_ARRAY, 1));
+                t.getRowsColumnRange(TEST_TABLE, ImmutableList.of(PtBytes.toBytes("row1")), BatchColumnRangeSelection.create(PtBytes.toBytes("col"), PtBytes.EMPTY_BYTE_ARRAY, 1));
```

Comment:
Maybe you want a separate scoped test for this, as opposed to putting this here?

Reply:
Fair enough, added one.